src/tfr_navigation/src/navigation_goal_manager.cpp

```diff
@@ -59,17 +64,17 @@ move_base_msgs::MoveBaseGoal NavigationGoalManager::initialize_goal() {
  */
 move_base_msgs::MoveBaseGoal NavigationGoalManager::get_updated_mining_goal(geometry_msgs::Pose msg)
 {
-    if (location_code == tfr_msgs::NavigationGoal::TO_MINING){
+    if (goal == tfr_msgs::LocationCode::MINING){
         nav_goal.target_pose.header.stamp =ros::Time::now();
-        //TODO integrate reference frame when bin tf publisher is finished
-        double y_position = msg.position.y; 
-        int sign = (y_position > 0) ? 1 : -1;
+        double v_position = msg.position.y; 
```

Comment:
Back to `v_position`?

Reply:
odd I don't know why that change didn't make it.